tests/summmarizer/test_image_description_retreiver.py

```diff
@@ -0,0 +1,66 @@
+import json
+from unittest.mock import Mock, patch
+
+from classifier_and_summarizer.summarizer.web_entity_detection import \
+    ImageDescriptionRetriever
+
```

Comment:
Add tests to verify the following as well:
- the request for web entity detection is formatted correctly
- error with correct description is returned when the server responds with an error code != 200

Reply:
Added the tests mentioned.